style.css

```diff
@@ -0,0 +1,100 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  background: #f1f1f1;
+}
+
+.main-header {
+  height: 50px;
+}
+
+.menu {
+  position: absolute;
+  left: 80%;
+  top: 4%;
+  font-size: 24px;
+}
+
+.main-header .menu {
+  left: 30;
+  line-height: 4;
+  display: flex;
+}
+
+.container {
+  background: #e5e5e5;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.part1 {
+  border: 1px solid rgb(202, 199, 199);
+  background: url(images/background.jpeg);
+  background-position: cover;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  display: flex;
+  flex-direction: column;
+  margin: 10px;
+  margin-bottom: 5%;
+  align-items: center;
+}
+
+.head-name {
+  font-family: 'Poppins', sans-serif;
+  font-size: 48px;
+  font-weight: 700;
+  margin: 40% 10px 17px;
+  line-height: 50px;
+  right: 10px;
+  display: flex;
+}
+
+.head-para {
+  font-family: 'Poppins', sans-serif;
+  left: 20px;
+  right: 10px;
+  margin: 10px 17px;
+}
+
+.sub-head {
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  font-size: 24px;
+  font-weight: 500;
+  margin-left: -48%;
+  color: #7f8cff;
+}
+
+.social-media ul {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  font-size: 20px;
+  padding: 0 10px;
+  margin-left: -30%;
+}
```

Comment:
- [ ] Please ensure that you imported the social links assets from the Figma design, since this milestone is about HTML and CSS, the project need to look identical to the design as much as possible 💯 

### Your Design
![Screenshot from 2023-03-21 23-51-46](https://user-images.githubusercontent.com/68030297/226759431-c3dbe6c8-fcf8-44e8-9ec9-743ebaa17970.png)

### Figma Design
![Screenshot from 2023-03-21 23-49-16](https://user-images.githubusercontent.com/68030297/226759343-6a7eba0b-5613-4bcf-946f-05cba7c4e909.png)